README.md

```diff
@@ -2,3 +2,64 @@
 
 A GitHub action uploads to the realm-template-apps s3 bucket. See [.github/workflows/zip-everything-and-upload-to-s3.yml](.github/workflows/zip-everything-and-upload-to-s3.yml).
 For bucket access, consult the Realm docs team.
+
+## Adding a New Template App
+
+1. Add your project in its own subdirectory.
+   If you are basing your project off of
+   an existing project (for example, creating a Flexible-Sync version of a
+   partition-based app), consider using the Bluehawk `state` tags and using
+   Bluehawk to copy each app to its own subdirectory within the `/generated`
+   directory.
+
+2. Be sure your project uses a `realm.json` file to get the app id and
+   base url info:
+
+   ```
+   {
+      "appId": "todo-sync-jxgjv",

```

Comment:
should this app Id be in the Readme? Or maybe just appId: "your-app-id"

Reply:
Supposed to be here for template apps. There's a script that parses things and updates the app id, too.